client/src/ui/molecules/document-survey/index.tsx

```diff
@@ -160,10 +160,13 @@ function SurveyDisplay({
     return null;
   }
 
+  const teaser = { __html: survey.teaser };
+  const question = { __html: survey.question };
+
   return (
     <div className="document-survey">
       <div className="survey-header">
-        <div className="survey-teaser">{survey.teaser}</div>
+        <div className="survey-teaser" dangerouslySetInnerHTML={teaser}></div>
```

Comment:
Instead of `dangerouslySetInnerHTML`, shouldn't we change the teaser/question type to `string | React.ReactNode`?